components/About/data.tsx

```diff
@@ -101,48 +101,43 @@ export const callsToAction = {
     {
       title: 'Hosting a node',
       content:
-        'Hosting a node using your graffiti automatically earns you points. You will only be rewarded points for hosting one node.',
-      points: ['12 hours = 100 points'],
+        'You can earn points by hosting an Iron Fish full node. To do so, make sure your telemetry is turned on, and your graffiti is set. You will only be rewarded points for hosting one node per graffiti.',
+      points: ['12 hours of continuous uptime = 100 points'],
       status: Status.New,
-      earn: 0,
-      disabled: true,
+      ctaText: 'Set up instructions',
+      href: 'https://coda.io/d/_dngSKECR9gO/Phase-2-FAQ_suBOW',
     },
     {
       title: 'Submit a Pull Request',
       content:
         'Submit a PR to the Iron Fish repo. Points are earned if the PR gets accepted and merged. Fill out the form to claim your points after your PR is merged.',
-      earn: 0,
       points: [
         'Small = 250 points',
         'Medium = 500 points',
         'Large = 1000 points',
       ],
       ctaText: 'Submit a PR',
-      // href: 'https://github.com/iron-fish/ironfish/pulls',
-      submissionForm: false,
+      href: 'https://github.com/iron-fish/ironfish/pulls',
```

Comment:
The text in this section mentions a form -- are we going to use the same form as last time? If so, we should switch this button to go to the form.

Reply:
Updated